src/designs/graph.html

```diff
@@ -6,6 +6,8 @@
     <meta name="description" content="Doo diddly doo I have no internet"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <link rel="stylesheet" href="./you-better-rename-this-later-tim.css">
+		
+		<link rel="icon" href="logos/favicon.svg">
```

Comment:
uau, svg favicons, so modern

Reply:
i am too lazy to make different sized ico files no thx